.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
```

Comment:
I decided to go with a separate CI workflow rather than overloading Netlify. Currently our Netlify build doesn't lint or run tests on the `react-showtime` module. We could do that by expanding on our build script and / or `package.json` hooks, but I don't think it provides feedback for a PR as clearly or offers as much configuration flexibility as GitHub Actions does. I would prefer keeping Netlify focused on deploying the static site, and Actions focused on testing and publishing the library.

Reply:
Sounds good.